Update x86_toolchain.sh

RidPanchal opened the pull request:
modified to improve arguments structure using 'getopts ' to handle command line options efficiently.

LubosKuzma commented:
Hi, please, provide me with a real name otherwise I can't merge this.
Thank you

Ipoduser16 commented:
Yes Daniel/Danny Williams Sent from my iPhoneOn Nov 30, 2023, at 12:36 PM, LubosKuzma ***@***.***> wrote:﻿
Hi, can you please provide your full name so I can properly grade this project? Thank you

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

LubosKuzma commented:
Thank you, Danny

RidPanchal committed:
Update x86_toolchain.sh

modified to improve arguments structure using 'getopts ' to handle command line options efficiently.
